### ISSUE TITLE
[BUG] container images are built without the commit SHA

### ISSUE THREAD

**Issue by maiqueb** [k8snetworkplumbingwg/multus-dynamic-networks-controller#75]
**Describe the bug**
When I check the commit SHA from a container image provided by this repo's package, I don't see the commit SHA from which the container image was built.

This `NONE` is the default value specified in the container [image](https://github.com/k8snetworkplumbingwg/multus-dynamic-networks-controller/blob/main/images/Dockerfile#L12).

**Expected behavior**
The command below should output the commit ID from which the container image was built.

**To Reproduce**
```bash
skopeo inspect docker://ghcr.io/k8snetworkplumbingwg/multus-dynamic-networks-controller:latest-amd64 -f '{{ index .Labels "multi.GIT_SHA" }}'
NONE
```

**Environment**:
- multus-dynamic-networks-controller version: N/A
- Kubernetes version (use `kubectl version`): N/A
- Network-attachment-definition: N/A
- OS (e.g. from /etc/os-release): N/A
- Controller configuration (`criSocketPath` / `multusSocketPath`): N/A
- Kernel (e.g. `uname -a`): N/A
- Others: N/A

**Additional info / context**
Add any other information / context about the problem here.

**Comment by maiqueb** [comment]
Failing w/ 
```
[Invalid workflow file: .github/workflows/image-push.yaml#L47](https://github.com/k8snetworkplumbingwg/multus-dynamic-networks-controller/actions/runs/3473197429/workflow)
The workflow is not valid. .github/workflows/image-push.yaml (Line: 47, Col: 13): A sequence was not expected
```
in this [github action](https://github.com/k8snetworkplumbingwg/multus-dynamic-networks-controller/actions/runs/3473197429).

**Comment by maiqueb** [comment]
Still not working ...
```bash
skopeo inspect docker://ghcr.io/k8snetworkplumbingwg/multus-dynamic-networks-controller:latest-amd64 -f '{{ index .Labels "multi.GIT_SHA" }}'
NONE
```

**Comment by oshoval** [comment]
Maybe worth to take a look here https://github.com/docker/build-push-action/issues/557 ?

**Comment by maiqueb** [comment]
Fixed in #80 .
